index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="/css/style.css" />
+        <title>ToDo App</title>
+    </head>
+    <body>
+        <header class="header">
+            <div class="header__container">
+                <nav class="header__nav">
+                    <div class="header__logo">
+                        <div class="header__logo-img">
+                            <img src="/images/leaf-1.png" alt="leaf" />
+                        </div>
+                        <div class="header__logo-text">ToDos</div>
+                    </div>
+                    <div class="header__search-input">
+                        <input type="text" placeholder="search todo" />
+                    </div>
+                </nav>
+            </div>
+        </header>
+        <section class="task-board">
+            <div class="task-board__container">
+                <h1 class="task-board__title">Add Tasks</h1>
+                <div class="task-board__button-area">
+                    <div class="task-board__button-area-left">
+                        <button class="task-board__button-create">
+                            Create Task
+                        </button>
+                    </div>
+                    <div class="task-board__button-area-right">
+                        <button class="task-board__button-filter">All</button>
```

Comment:
I see the classnames are same, can't we just use the classname of parent div and add style on each button?

Reply:
i try to ARRANGE the class name again and need to review again